src/graph/graph.people.ts

```diff
@@ -45,13 +45,13 @@ export async function findPeople(
   graph: IGraph,
   query: string,
   top: number = 10,
-  personType: PersonType = PersonType.Person
+  personType: PersonType = PersonType.person
 ): Promise<Person[]> {
   const scopes = 'people.read';
 
   let filterQuery = '';
 
-  if (personType !== PersonType.Any) {
+  if (personType !== PersonType.any) {
     filterQuery = `personType/class eq '${personType}'`;
```

Comment:
I think this might need to capitalized now that I think about it. The pesonType/class are capitalized (ie `Person`, `Group`)

Reply:
conversion here:
22edcc3